docs/allow-nested-if-parent-exist.md

```diff
@@ -0,0 +1,56 @@
+# AllowNestedIfParentExist
+
+## Category
+
+ARM Error
+
+## Applies to
+
+ARM OpenAPI(swagger) specs
+
+## Related ARM Guideline Code
+
+- RPC-Get-V1-11
+
+## Description
+
+When the parent resource is scoped under a resource group, the path for the list calls of its child resources must also be scoped under a resource group
+
+## How to fix
+
+To fix this, ensure that the LIST call for the child resource has a resource group segment.
+
+### Invalid example
+
+Following are the patterns for the invalid examples
+GET /subscriptions/{sub1}/providers/{ProviderName}/resourceType1/{resourceTypeName}/nestedResourceType
+GET /subscriptions/{sub1}/providers/{ProviderName}/nestedResourceType
+GET /subscriptions/{sub1}/resourcegroups/{rgname}/providers/{ProviderName}/nestedResourceType
+
+```json
+"paths": {
+  "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ProviderNamespace/ParentResourceType/{parentInstance}/NestedResourceType": {
+    "get": {
+        //Nested Resource Type Get Call definition
+    },
+  }
+// No Parent Resource Get Call definied
+}
+```
```

Comment:
The way this is documented currently is hard to understand. Consider something like this

Invalid Examples

Example 1: In this example, the nested resource type's List operation does not have resourceGroups segment whereas its parent has the resourceGroups section.

```json

API path of the parent resource type's get operation
"paths": {
  "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ProviderNamespace/ParentResourceType/{parentInstance}": {
    "get": {
      ...
    },
  }
}

API path of the nest resource type's List operation
"paths": {
  "/subscriptions/{subscriptionId}/providers/Microsoft.ProviderNamespace/ParentResourceType/{parentInstance}/NestedResourceType": {
    "get": {
      ...
    },
  }
}


Add more examples in this format